LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -322,6 +327,55 @@ extension TherapySettingsView {
     }
 }
 
+// MARK: Navigation
+
+extension TherapySettingsView {
```

Comment:
this was removed from here and put in the view model. Curious why it is being put back?

Reply:
I don't know the history of why it was originally in the view itself and then moved to the view model. However, both the mode and color palette were moved to the view to allow for appropriate initialization of some of the therapy settings views when injected into the onboarding process. This basically came down to an issue of glueing these views into onboarding after other SwiftUI views. (It worked fine as it was when all of these views were managed by a UIKit "coordinator", but we could not use the coordinator pattern any longer because it ended up trying to display a `NavigationController` within a `NavigationController`, which lead to no end of nav bar issues. Thus, the entire Your Settings workflow is now all SwiftUI with no intermediary UIKit coordinator.)